docs/advanced/types-react-ap.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: types_react_api
-title: Section 4: @types/react and @types/react-dom APIs
-sidebar_label:  @types/react and @types/react-dom APIs
+title: "Section 4: @types/react and @types/react-dom APIs"
```

Comment:
TIL how to use colons in front matter! 🎉

Reply:
IMHO this is specific to graymatter and one of the error I noted to improve because currently a : in frontmatter produces a quite cryptic error message

Reply:
Reply